test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -10847,12 +10866,12 @@ def forward(self, x):
         model.qconfig = torch.quantization.get_default_qconfig("fbgemm")
         model = torch.quantization.prepare_qat(model)
         # Set fixed weight and bias to avoid flaky test.
-        model.conv.weight = torch.nn.Parameter(torch.arange(4 * 2 * 3 * 3, dtype=torch.float).view(2, 4, 3, 3) / 36)
-        model.conv.bias = torch.nn.Parameter(torch.arange(2, dtype=torch.float))
+        model.conv.weight = torch.nn.Parameter(_construct_tensor_for_quantization_test((2, 4, 3, 3), max_val=2))
+        model.conv.bias = torch.nn.Parameter(torch.tensor([0., 1.]))
```

Comment:
nit: Prefer adding the `0` for making a number float. `0.0` makes it apparent to readers that the number is a float.

Reply:
is there a case where `0.` could be not floating number? I feel the current style is equally readable while being less verbose.

Reply:
0. is perfectly fine syntactically. I am fine with both ways and in fact wrote it like so before :). I received this suggestion in a past readability review so I was passing it on.

Reply:
Discussions are always welcomed :). For this particular case I think both ways are fine. The `0.` approach makes tensors/arrays more compact which is in my opinion better for displaying larger tensors in code, hence followed as convention.

Reply:
SGTM!

Reply:
+1 for `0.`.